mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -2598,6 +2598,107 @@ fn test_get_max_taker_vol_with_kmd() {
     block_on(mm_alice.stop()).unwrap();
 }
 
+#[test]
+fn test_get_max_maker_vol() {
+    let (_ctx, _, priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN1", 1.into());
+    let coins = json!([mycoin_conf(1000), mycoin1_conf(1000)]);
+    let mm = MarketMakerIt::start(
+        json!({
+            "gui": "nogui",
+            "netid": 9000,
+            "dht": "on",  // Enable DHT without delay.
+            "passphrase": format!("0x{}", hex::encode(priv_key)),
+            "coins": coins,
+            "rpc_password": "pass",
+            "i_am_seed": true,
+        }),
+        "pass".to_string(),
+        None,
+    )
+    .unwrap();
+    let (_dump_log, _dump_dashboard) = mm_dump(&mm.log_path);
+
+    log!("{:?}", block_on(enable_native(&mm, "MYCOIN", &[])));
+    log!("{:?}", block_on(enable_native(&mm, "MYCOIN1", &[])));
+    let rc = block_on(mm.rpc(&json!({
+        "userpass": mm.userpass,
+        "mmrpc": "2.0",
+        "method": "max_maker_vol",
+        "params": {
+            "coin": "MYCOIN1",
+        }
+    })))
+    .unwrap();
+    assert!(rc.0.is_success(), "!max_taker_vol: {}", rc.1);
```

Comment:
Please add `max_maker_vol` helper to `for_tests` instead. As `structs.rs` were moved to `mm2_test_helpers` recently, it can return `MaxMakerVolResponse` right away, similar to https://github.com/KomodoPlatform/atomicDEX-API/blob/593032cf869b1cfee9f91fc3df758bbbadb5f485/mm2src/mm2_test_helpers/src/for_tests.rs#L2306